changelog/fragments/phase-2-plugin-keys.yaml

```diff
@@ -0,0 +1,22 @@
+entries:
+  - description: >
+      (ansible/v1, helm/v1, go/v2) Added plugin keys for the `manifests` and `scorecard` phase 2 plugins
+    kind: addition
+    breaking: false
+    migration:
+      header: Add `manifests` and `scorecard` plugin keys to your PROJECT
```

Comment:
```suggestion
      header: (For Ansible/Helm based-operators) Update the PROJECT file to add the SDK plugins
```

Reply:
This applies to all plugin types.

Reply:
Why it applies to all plugins? The golang one has these entries already. See: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/PROJECT#L12-L14. So, a user with golang based-operator project would need to follow it? So, we are not changing or add anything to go. Am I right?

Reply:
I catcher: https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v2/memcached-operator/PROJECT#L10-L11 (go/v2) does not do the same as v3.